pennylane/capture/__init__.py

```diff
@@ -40,5 +40,78 @@
     >>> qml.capture.enabled()
     False
 
+**Custom Operator Behaviour**
+
+Any operator that inherits from :class:`~.Operator` gains a default ability to be captured
+in a Jaxpr. Any positional argument is bound as a tracer, wires are processed out into individual tracers,
+and any keyword arguments are passed as keyword metadata.
+
+.. code-block:: python
+
+    class MyOp1(qml.operation.Operator):
+
+        def __init__(self, arg1, wires, key=None):
+            super().__init__(arg1, wires=wires)
+
+    def qfunc(a):
+        MyOp1(a, wires=(0,1), key="a")
+
+    qml.capture.enable()
+    print(jax.make_jaxpr(qfunc)(0.1))
+
+.. code-block::
+
+    { lambda ; a:f32[]. let
+        _:AbstractOperator() = MyOp1[key=a n_wires=2] a 0 1
+    in () }
+
+But an operator developer may need to override custom behavior for calling ``cls._primitive.bind`` 
+(where ``cls`` indicates the class) if:
+
+* The operator does not accept wires, like :class:`~.SymbolicOp` or :class:`~.CompositeOp`.
+* The operator allows metadata to be provided positionally, like :class:`~.PauliRot`.
+
+In such cases, the operator developer can override ``cls._primitive_bind_call``.  This is what
+will be called when constructing a new class instance instead of ``type.__call__``.  For example,
```

Comment:
```suggestion
In such cases, the operator developer can override ``cls._primitive_bind_call``, which
will be called when constructing a new class instance instead of ``type.__call__``.  For example,
```